.github/workflows/scripts/govulncheck-run.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mkdir -p ./govulncheck 2>/dev/null
+
+# Get all package directories
+ALL_PKG_DIRS=$(go list ./...)
+
+# Initialize failure flag
+FAILED=0
+
+# Repository prefix to remove from package names
+REPO_PREFIX=$(go list -m)
+
+# Run govulncheck for each package
+for pkg in $ALL_PKG_DIRS; do
+  OUTPUT_FILE="./govulncheck/$(echo "$pkg" | sed "s|^$REPO_PREFIX/||" | tr '/' '_').sarif"
```

Comment:
Can you add a comment explaining what `sed` and `tr` are doing here?

Reply:
f242a1a43fa30102455de2803c4cd86b62db39e6